sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsServiceVersion.java

```diff
@@ -10,7 +10,8 @@
  */
 public enum TextAnalyticsServiceVersion implements ServiceVersion {
     V3_0("v3.0"),
-    V3_1_PREVIEW_1("v3.1-preview.1");
+    V3_1_PREVIEW_1("v3.1-preview.1"),
```

Comment:
before we GAd, did we use to have a list of all the preview versions we've supported?

Reply:
Good point. I can just keep the latest preview version instead of keep all.